pom.xml

```
@@ -143,6 +143,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-picocontainer</artifactId>
```

```suggestion
            <artifactId>cucumber-picocontainer</artifactId>
            <version>${io.cucumber.version}</version>
```

all the other cucumber dependencies have a version by now, i think we should also use it here.